extensions/machine-learning-services/src/modelManagement/registeredModelService.ts

```diff
@@ -56,22 +95,73 @@ export class RegisteredModelService {
 
 	private async runRegisteredModelsListQuery(connection: azdata.connection.ConnectionProfile): Promise<azdata.SimpleExecuteResult | undefined> {
 		try {
-			return await this._queryRunner.runQuery(connection, this.registeredModelsQuery(this._config.registeredModelDatabaseName, this._config.registeredModelTableName));
+			return await this._queryRunner.runQuery(connection, this.registeredModelsQuery(connection.databaseName, this._config.registeredModelDatabaseName, this._config.registeredModelTableName));
+		} catch {
+			return undefined;
+		}
+	}
+
+	private async runUpdateModelQuery(connection: azdata.connection.ConnectionProfile, model: RegisteredModel): Promise<azdata.SimpleExecuteResult | undefined> {
+		try {
+			return await this._queryRunner.runQuery(connection, this.getUpdateModelScript(connection.databaseName, this._config.registeredModelDatabaseName, this._config.registeredModelTableName, model));
 		} catch {
 			return undefined;
 		}
 	}
 
-	private registeredModelsQuery(databaseName: string, tableName: string) {
+	private registeredModelsQuery(currentDatabaseName: string, databaseName: string, tableName: string): string {
+		if (!currentDatabaseName) {
+			currentDatabaseName = 'master';
+		}
+		return `
+		${this.configureTable(databaseName, tableName)}
+		USE ${currentDatabaseName}
+		SELECT artifact_id, artifact_name, name, description, version, created from ${databaseName}.dbo.${tableName}
+		WHERE artifact_name not like 'MLmodel' and artifact_name not like 'conda.yaml'
+		Order by artifact_id
+		`;
+	}
+
+	private configureTable(databaseName: string, tableName: string): string {
 		return `
-		IF (EXISTS (SELECT name
-			FROM master.dbo.sysdatabases
-			WHERE ('[' + name + ']' = '${databaseName}'
-			OR name = '${databaseName}')))
+		USE ${databaseName}
+		IF EXISTS
+			(  SELECT [name]
+				FROM sys.tables
+				WHERE [name] = '${tableName}'
+			)
 		BEGIN
-			SELECT artifact_id, artifact_name, group_path, artifact_initial_size from ${databaseName}.${tableName}
-			WHERE artifact_name like '%.onnx'
+			IF NOT EXISTS (SELECT * FROM syscolumns WHERE ID=OBJECT_ID('${tableName}') AND NAME='name')
```

Comment:
>tableName [](start = 65, length = 9)

tableName needs to be escaped properly in all the places in this query - you have a mix of single quote and bracket quoting so use the appropriate escaping in each instance.